config/crd/common/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+# This file is NOT auto-generated
```

Comment:
why not?

Reply:
This is copied from the code in `runtime`. Don't blame @A-Hilaly for this :P

Reply:
Ah, OK. I blame you then, @RedbackThomson :)

Reply:
Let's just blame @ack-bot :smirk: